recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiohttp_client_cache-{{ version }}.tar.gz
```

Comment:
Oh weird. So the change that broke automerge here was redirect behavior on pypi.io? Previously it would normalize package names (underscores vs. dashes) in the artifact URL. Thanks for fixing that.

Reply:
It's possible, not sure though.